### PR TITLE
pretty print trace error

### PR DESCRIPTION
**Pull request by debris**
fixes https://github.com/ethcore/parity/issues/1052

### CODE REVIEW

File: ethcore/src/trace/error.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! TraceDB errors.
+
+use std::fmt::{Display, Formatter, Error as FmtError};
+
+const RESYNC_ERR: &'static str =
+"Your current parity installation has synced without transaction tracing. To use
+Parity with transaction tracing, you'll need to resync with tracing. To do this, remove or move away your current
```
Comment (gavofyork) [inline ethcore/src/trace/error.rs]: weird line breaking?

**Comment by debris** [comment]
conflicts resolved

**Comment by arkpar** [comment]
does not compile

**Comment by debris** [comment]
I merged incorrectly (do not know how it happened). Should be fine now. I will remove gotiisues flag once travis is green

**Pull request merged**